src/main/resources/com/nirima/jenkins/plugins/docker/DockerCloud/help-containerCapStr.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The maximum number of containers that this provider is allowed to have running, defaults to blank which disables the limit
```

Comment:
Maybe shorten to "is allowed to run" and "blank - unlimited". Also what will happen if i enter 0 ?